moped-editor/src/utils/activityLogFormatters/mopedComponentsActivity.js

```diff
@@ -44,16 +44,13 @@ export const formatComponentsActivity = (
   if (change.description.length === 0) {
     return {
       changeIcon,
-      changeText: [
-        { text: "Added a component: ", style: null },
-        componentText,
-      ],
+      changeText: [{ text: "Added a component: ", style: null }, componentText],
     };
   }
 
   // delete an existing component
-  if (change.description[0].field === "is_deleted") {
-    const {link, ...simpleComponentText} = componentText;
+  if (change.description[0].fields.includes("is_deleted")) {
```

Comment:
This is the fix across all these record types. `field` turned into an array called `fields` that can include more than one column name. I previously thought that this only affected project summary activities but I was wrong and am fixing it.